src/components/Menu/Menu.tsx

```diff
@@ -46,16 +46,6 @@ export interface MenuProps extends UIComponentProps<any, any>, ChildrenComponent
   /** The menu can have primary type. */
   primary?: boolean
 
-  /**
-   * A custom render iterator for rendering each of the Menu items.
-   * The default component, props, and children are available for each item.
-   *
-   * @param {React.ReactType} Component - The computed component for this slot.
-   * @param {object} props - The computed props for this slot.
-   * @param {ReactNode|ReactNodeArray} children - The computed children for this slot.
-   */
-  renderItem?: ShorthandRenderFunction
```

Comment:
Same as the Accordion, I know that the user can use the current API and mapping logic for introducing this, but notice that this was not added with the renderXXX methods PR, it was here before, because it introduces clearer API for rendering each item in the Menu.